### PR TITLE
Rescope classes

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-16722

- Add "adv" prefix to classes
- Use PF/Platform class naming convention
- general cleanup

Screen Shot 2021-12-07 at 9 24 39 AM
Screen Shot 2021-12-07 at 9 25 18 AM
Screen Shot 2021-12-07 at 9 25 01 AM
Screen Shot 2021-12-07 at 9 24 51 AM

### CODE REVIEW

File: src/PresentationalComponents/Cards/Pathways.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const PathwayCard = (props) => {
     <Card
       isFlat
       isPlain
-      className={`ins-c-advisor__card--pathwaycard advisor__background--global-100`}
+      className={`adv-c-card-pathway adv__background--global-100`}
```
Comment (Hyperkid123) [inline src/PresentationalComponents/Cards/Pathways.js:54]: This string does not have to be wrapped in `{}`
```
className="adv-c-card-pathway adv__background--global-100"
```

**Review by Hyperkid123 — COMMENTED** [review]
Overall looks good. One thing I would change though are some of the `className` props as they can be just written as plain string values
```diff
-className={`adv-c-card-pathway adv__background--global-100`}
+className="adv-c-card-pathway adv__background--global-100"
```

It's used like that in multiple places.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by karelhala** [comment]
Ping @RedHatInsights/team-interact code looks good, I just want you to know about this change so you can see if there are any CSS anomalies (and preferably merge on your own pace)

File: src/PresentationalComponents/RuleDetails/RuleDetails.js [hunk 2]
```diff
@@ -210,10 +210,8 @@ const BaseRuleDetails = ({
                   <StackItem>
                     <strong>{intl.formatMessage(messages.riskofchange)}</strong>
                   </StackItem>
-                  <StackItem
-                    className={`pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm`}
-                  >
-                    <span className="ins-c-rule-details__stackitem">
+                  <StackItem className="pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm">
```
Comment (mkholjuraev) [inline src/PresentationalComponents/RuleDetails/RuleDetails.js:213]: Should we change **alignCenterOverride** to comply with this PR?

Comment (epwinchell) [reply src/PresentationalComponents/RuleDetails/RuleDetails.js:213]: @mkholjuraev Yes, thanks!

**Pull request merged**